builderConfig.js

```diff
@@ -0,0 +1,125 @@
+const env = process.env.NODE_ENV || 'production';
+const isBuildingDev = env.startsWith( 'dev' );
+
+const buildConfig = {
+    productName: 'SAFE Browser',
+    files: [
+        'app/dist/',
+        'locales',
+        'app/locales',
+        'app/app.html',
+        'app/bg.html',
+        'app/background.prod.js',
+        'app/background.prod.js.map',
+        'app/webPreload.prod.js',
+        'app/webPreload.prod.js.map',
+        'app/main.prod.js',
+        'app/main.prod.js.map',
+        'package.json'
+    ],
+    extraFiles: [],
+    extraResources: [
+        {
+            from: 'resources/log.toml',
+            to: 'log.toml'
+        },
+        {
+            from: 'resources/favicon.ico',
+            to: 'favicon.ico'
+        },
+        {
+            from: 'resources/PreloadDevVault',
+            to: 'PreloadDevVault'
+        },
+        {
+            from: 'resources/locales',
+            to: 'locales'
+        },
+        {
+            from: 'resources/crust.config',
+            // eslint-disable-next-line no-template-curly-in-string
+            to: '${productName}.crust.config'
+        },
+        {
+            from: 'app/extensions/safe/dist',
+            to: 'extensions/safe/dist'
+        },
+        {
+            from: 'app/extensions/safe/iconfont',
+            to: 'extensions/safe/iconfont'
+        },
+        {
+            from: 'app/extensions/safe/auth-web-app/dist',
+            to: 'extensions/safe/auth-web-app/dist'
+        },
+        {
+            from: 'node_modules/nessie-ui/dist/styles.css',
+            to: 'nessie-styles.css'
+        }
+    ],
+    dmg: {
+        contents: [
+            {
+                x: 130,
+                y: 220
+            },
+            {
+                x: 410,
+                y: 220,
+                type: 'link',
+                path: '/Applications'
+            }
+        ],
+        artifactName: `\${name}-\${version}-\${arch}-${env}.\${ext}`
+    },
+    protocols: {
+        name: 'SAFE Network URL',
+        schemes: ['safe', 'safe-auth']
+    },
+    win: {
+        target: ['nsis', 'zip'],
+        extraFiles: [
+            {
+                from: 'resources/crust.config',
+                // eslint-disable-next-line no-template-curly-in-string
+                to: '${productName}.crust.config'
+            }
+        ]
+    },
+    nsis: {
+        artifactName: `\${name}-\${version}-\${arch}-${env}.\${ext}`
+    },
+    linux: {
+        target: ['AppImage', 'zip'],
+        category: 'Development',
+        extraFiles: [
+            {
+                from: 'resources/crust.config',
+                // eslint-disable-next-line no-template-curly-in-string
+                to: '${name}.crust.config'
```

Comment:
could this not be a template string and therefor not need this eslint evasion? ie:
```js
`${name`.crust.config`
```